source/Ship.cpp

```diff
@@ -2640,7 +2653,11 @@ int Ship::TakeDamage(const Projectile &projectile, bool isBlast)
 	else if(shieldDamage > shields)
 		shieldFraction = min(shieldFraction, shields / shieldDamage);
 	shields -= shieldDamage * shieldFraction;
+	if(shieldDamage)
+		shieldDelay = (shields <= 0.) ? attributes.Get("disabled shield delay") : attributes.Get("shield delay");
```

Comment:
`"disabled shield delay"` applies even if the ship is not actually disabled?
I would wait to set the delays until you know if the ship is disabled or not, and then you can make sure that the disabled hull delay is set for all hull damage received after being disabled and not just the initial hit that disabled the ship.

Reply:
I've reworded the tooltips to clarify the conditions that each delay is suppose to trigger.